### PR TITLE
add ca-certificates package to enable verifying lets encrypt certs

### PR DESCRIPTION
**Pull request by gurvindersingh**
## Description
In current docker image, it is not possible to connect to Clickhouse which has let's encrypt certificates. It fails with this error

```
ERROR: 2023/02/15 13:13:16 scope.go:699: error while health-checking "XXX.example.com" host: cannot send request in 30.873573ms: Get "XXX.example.com/?query=SELECT%201%2B1": x509: certificate signed by unknown authority
```

This PR fixes it by adding `ca-certificates` package into base image.

 

## Pull request type
Updated docker image
 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ x ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Checklist

- [ x  ] Linter passes correctly
- [ ] Add tests which fail without the change (if possible)
- [ x ] All tests passing
- [ ] Extended the README / documentation, if necessary

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by Blokje5 — APPROVED** [review]
Thanks for your contribution! Once CI passes I will merge this PR.

**Comment by gurvindersingh** [comment]
Thanks @Blokje5 for merging it quickly, any plan to cut a release so that I can have the updated docker image for deployment 🙏🏽

**Comment by LEijsackersCS** [comment]
@gurvindersingh I'll discuss it with the other maintainers soon. We should probably decide on a release schedule so it is clear when everyone can expect the next set of features & fixes.

**Comment by Blokje5** [comment]
@gurvindersingh I'll discuss it with the other maintainers soon. We should probably decide on a release schedule so it is clear when everyone can expect the next set of features & fixes.

**Pull request merged**
